projects/pastanaga-angular/src/lib/modal/dialog/dialog.component.scss

```diff
@@ -0,0 +1,32 @@
+@import "../../styles/variables";
```

Comment:
`@import "variables";` should be good enough

Reply:
it would work but I prefer not using it in pastanaga lib because it requires an extra configuration on project using pastanaga to make it works